editor/property_editor.cpp

```diff
@@ -4514,7 +4514,12 @@ void SectionedPropertyEditor::update_category_list() {
 		else if (!(pi.usage & PROPERTY_USAGE_EDITOR))
 			continue;
 
-		if (pi.name.find(":") != -1 || pi.name == "script" || pi.name == "resource_name" || pi.name == "resource_path" || pi.name == "resource_local_to_scene")
+		if (pi.name.find(":") != -1 ||
+				pi.name == "script" ||
+				pi.name == "resource_name" ||
+				pi.name == "resource_path" ||
+				pi.name == "resource_local_to_scene" ||
+				pi.name.begins_with("interface/dialogs"))
```

Comment:
If it's not meant to be edited by the user, it shouldn't be saved as editor settings IMO.

Reply:
It's indirectly editable by moving and resizing windows. Would you still keep it then?
Editing the editor_settings_bounds while it's open doesn't work anyway and will be overriden when the window is closed.

Reply:
Better remove that hardcoded skip (the other ones are related to the nature of PropertyEditor, not to the editor settings) for this PR, we can discuss how to handle those separately.

Reply:
I wanted to use some special PROPERTY_USAGE_ for those properties (like EDITOR but HIDDEN), but didn't find one. That would've been better than hardcoding it, I guess. I remove it for now.

Reply:
I think that non-editable editor metadata should be saved in a file different from editor-settings-3.tres.

Reply:
Is that possible atm and are there places where it has already been done (so I can steal code)? I created an issue to keep track of it.